src/app/modules/payments/stripe.service.ts

```diff
@@ -559,7 +559,7 @@ export const handleStripeEvent = (
             { stripeAccount: event.account },
           )
           .autoPagingEach(async (balanceTransaction) => {
-            if (balanceTransaction.type !== 'charge') return
```

Comment:
nit: sudden thought. Might be useful to log scenarios when the early return happens